rules/send-email-tasks-to-individuals/rule.md

```diff
@@ -7,27 +7,29 @@ authors:
     url: https://www.ssw.com.au/people/uly
 created: 2022-03-15T18:31:01.031Z
 guid: 4b0321a9-a4fe-4f60-a9e2-dd0b764cd1bc
+related:
+  - include-names-as-headings
 
 ---
 
-It’s common to send email tasks to a whole team or multiple team members, but that is not a good idea. They may all assume that someone else will do the task, and ignore it. 
+Sending email tasks to a whole team or multiple team members is not a good idea because it can lead to confusion and lack of accountability. When everyone is responsible for a task, it can be easy for each person to assume that someone else will take care of it, leading to the task being ignored or forgotten.
+
+Instead, it is recommended to assign tasks to specific individuals and make sure that each person knows what they are responsible for. This can help ensure that tasks are completed on time and that everyone is held accountable for their work .
```

Comment:
```suggestion
Instead, it is recommended to assign tasks to specific individuals and make sure that each person knows what they are responsible for. This can help ensure that tasks are completed on time and that everyone is held accountable for their work.
```